sdk/storage/storage-blob/rollup.base.config.js

```diff
@@ -64,14 +64,18 @@ export function nodeConfig(test = false) {
 
   if (test) {
     // entry point is every test file
-    baseConfig.input = ["dist-esm/test/*.spec.js", "dist-esm/test/node/*.spec.js", "dist-esm/src/index.js"];
+    baseConfig.input = [
+      "dist-esm/test/*.spec.js",
+      "dist-esm/test/node/*.spec.js",
+      "dist-esm/src/index.js"
+    ];
     baseConfig.plugins.unshift(multiEntry());
 
     // different output file
     baseConfig.output.file = "dist-test/index.node.js";
 
     // mark assert as external
```

Comment:
should we update this comment since it's not just assert that is getting excluded now?